qa/SidechainTestFramework/account/smart_contract_resources/.npmignore

```diff
@@ -0,0 +1,3 @@
+hardhat.config.ts
+scripts
+test
```

Comment:
Afaik `.npmignore` is only relevant when publishing NPM packages. We won't do that here, so remove it.